src/libs/error.mock.ts

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash/fp'
+
+/**
+ * Allows observation of when an error occurs via `errorWatcher`, i.e.:
+ *     expect(errorWatcher).toHaveBeenCalledTimes(1)
+ *     expect(errorWatcher).toHaveBeenCalledWith('my error message', expect.anything())
+ * You must use `jest.resetMocks` in `beforeEach` to clear the errorWatcher between tests
```

Comment:
Jest is configured to clear mocks for the project.... so this comment should be tweaked a bit:

"The errorWatcher call counts and information will be cleared in between tests thanks to Jest being configured to auto-clear mock between tests at the project level."